content/source/docs/cloud/sentinel/examples.html.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: "cloud"
-page_title: "Example Policies - Sentinel - Terraform Cloud"
+page_title: "Example Policies - Policies - Terraform Cloud"
 ---
 
 # Example Policies
 
--> **Note:** Sentinel policies are a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
+-> **Note:** Terraform policies are a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
 
-This page lists some example Sentinel policies. These examples are not exhaustive, but they demonstrate some of the most common use cases of Sentinel with Terraform Cloud. For more examples, see the [Governance section of the Terraform Guides repository](https://github.com/hashicorp/terraform-guides/tree/master/governance).
+This page lists some example policies. These examples are not exhaustive, but they demonstrate some of the most common use cases of Policies with Terraform Cloud. For more examples, see the [Governance section of the Terraform Guides repository](https://github.com/hashicorp/terraform-guides/tree/master/governance).
 
-~> **Important:** These examples are a demonstration of Sentinel's features. They should not be used verbatim in your Terraform Cloud organization. Make sure you fully understand the intent and behavior of a policy before relying on it in production.
+~> **Important:** These examples are a demonstration of the Sentinel policy language features. They should not be used verbatim in your Terraform Cloud organization. Make sure you fully understand the intent and behavior of a policy before relying on it in production. 
```

Comment:
```suggestion
~> **Important:** These examples are a demonstration of the Sentinel policy language and its features. They should not be used verbatim in your Terraform Cloud organization. Make sure you fully understand the intent and behavior of a policy before relying on it in production. 
```

I can't read the other sentence, but it might just be me.